external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DB2IntegrationSuite.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.tags.DockerTest
 @Ignore // AMPLab Jenkins needs to be updated before shared memory works on docker
 class DB2IntegrationSuite extends DockerJDBCIntegrationSuite {
   override val db = new DatabaseOnDocker {
-    override val imageName = "lresende/db2express-c:10.5.0.5-3.10.0"
+    override val imageName = "ibmcom/db2:11.5.0.0"
```

Comment:
This image comes with the IBM® Db2® Developer-C edition license pre-applied and is a full feature, free version for production and non-production environments:
https://hub.docker.com/r/ibmcom/db2